TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/EditInduction/StartDate.cshtml.cs

```diff
@@ -23,19 +35,36 @@ public string BackLink
         get => PageLink(InductionJourneyPage.Status);
     }
 
-    public StartDateModel(TrsLinkGenerator linkGenerator) : base(linkGenerator)
+    public StartDateModel(TrsLinkGenerator linkGenerator, TrsDbContext dbContext, IClock clock) : base(linkGenerator)
     {
+        _dbContext = dbContext;
+        _clock = clock;
     }
 
     public void OnGet()
     {
+        StartDate = JourneyInstance!.State.StartDate;
     }
 
     public async Task<IActionResult> OnPostAsync()
     {
+        if (StartDate > _clock.Today)
+        {
+            ModelState.AddModelError(nameof(StartDate), "The induction start date cannot be in the future");
+        }
+        if (StartDate < new DateOnly(1999, 5, 7)) // CML TODO - business logic - move
+        {
+            ModelState.AddModelError(nameof(StartDate), "The induction start date cannot be before 7th May 1999");
+        }
```

Comment:
Validation logic to be added to `person` ? Not sure I can add it to the `ValidateInductionData` method because that's being used for updating the cpd induction data which I guess might have dates prior to 7th may 1999?

Reply:
We could put a `staticDateOnly EarliestInductionStartDate` on `Person`; we don't have a great way of sharing validation logic with Razor Pages currently.